src/api/spec/components/previews/bs_request_action_description_component_preview.rb

```diff
@@ -1,12 +1,12 @@
 class BsRequestActionDescriptionComponentPreview < ViewComponent::Preview
   # Preview at http://HOST:PORT/rails/view_components/bs_request_action_description_component/submit_preview
   def submit_preview
-    bs_request = BsRequest.joins(:bs_request_actions).where(bs_request_actions: { type: :submit }).last
-    render(BsRequestActionDescriptionComponent.new(action: bs_request.bs_request_actions.last))
+    action = BsRequestAction.where(type: :submit).last
```

Comment:
This would be a random `BsRequestActionSubmit` most likeley unrelated to this `BsRequest`

Reply:
There is no `this BsRequest`... The component renders an action, that's what I query for....

Reply:
ah this is for preview. okay...